css/.sass-lint.yml

```diff
@@ -0,0 +1,350 @@
+        - padding-left
+        - border
+        - border-width
+        - border-style
+        - border-color
+        - border-top
+        - border-top-width
+        - border-top-style
+        - border-top-color
+        - border-right
+        - border-right-width
+        - border-right-style
+        - border-right-color
+        - border-bottom
+        - border-bottom-width
+        - border-bottom-style
+        - border-bottom-color
+        - border-left
+        - border-left-width
+        - border-left-style
+        - border-left-color
+        - null
+        - visibility
+        - opacity
+        - border-radius
+        - border-top-left-radius
+        - border-top-right-radius
+        - border-bottom-right-radius
+        - border-bottom-left-radius
+        - border-image
+        - border-image-source
+        - border-image-slice
+        - border-image-width
+        - border-image-outset
+        - border-image-repeat
+        - background
+        - background-color
+        - background-image
+        - background-repeat
+        - background-attachment
+        - background-position
+        - background-position-x
+        - background-position-y
+        - background-origin
+        - background-size
+        - background-clip
+        - box-shadow
+        - box-decoration-break
+        - outline
+        - outline-width
+        - outline-style
+        - outline-color
+        - filter
+        - null
+        - color
+        - font
+        - font-family
+        - font-style
+        - font-weight
+        - font-variant
+        - font-size
+        - font-size-adjust
+        - line-height
+        - letter-spacing
+        - word-spacing
+        - direction
+        - word-wrap
+        - overflow-wrap
+        - word-break
+        - hyphens
+        - white-space
+        - text-align
+        - text-align-last
+        - text-indent
+        - text-overflow
+        - text-transform
+        - text-decoration
+        - text-shadow
+        - text-rendering
+        - text-size-adjust
+        - vertical-align
+        - tab-size
+        - list-style
+        - list-style-position
+        - list-style-type
+        - list-style-image
+        - null
+        - transform
+        - transform-function
+        - transform-origin
+        - transform-style
+        - perspective
+        - perspective-origin
+        - backface-visibility
+        - null
+        - transition
+        - transition-property
+        - transition-duration
+        - transition-timing-function
+        - transition-delay
+        - animation
+        - animation-name
+        - animation-duration
+        - animation-timing-function
+        - animation-delay
+        - animation-iteration-count
+        - animation-direction
+        - animation-fill-mode
+        - animation-play-state
+        - null
+        - cursor
+        - user-select
+        - resize
+        - pointer-events
+        - null
```

Comment:
I suspect these Null values will need to be removed. I'm fairly certain they don't add spaces between properties groups.